test/regression.js

```diff
@@ -18,11 +19,21 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const width = 960;
 const height = 720;
 
+const FILE_PATTERN = new RegExp('.*');
```

Comment:
It'd be much simpler to just catch an arg and handle this when collecting files.

```diff
    try {
      const start = process.hrtime.bigint();
      const fixturesDir = path.join(__dirname, 'regression-fixtures');
      const filesPromise = fs.readdir(fixturesDir, { recursive: true });
+     const patternArg = process.argv[2];
+     const regex = (patternArg) ? new RegExp(patternArg) : null;
…
-     const list = (await filesPromise).filter((name) => name.endsWith('.svg'));
+     const list = (await filesPromise)
+       .filter((name) => name.endsWith('.svg'))
+       .filter((name) => regex === null || regex.test(name));
```

Usage:

* `node test/regression color-prop`
* `yarn run test-regression color-prop`